roles/mapping/tasks/main.yml

```diff
@@ -7,10 +7,10 @@
   tags:
     - aur
 
-- name: Install OpenCPN
-  aur: name=opencpn user={{ user.name }}
-  tags:
-    - aur
+#- name: Install OpenCPN
```

Comment:
We're being inconsistent between commenting tasks out like this and adding architecture conditionals. I don't have a strong preference between the two methods, but I think keeping it consistent it everywhere will keep our future lives easier.

The conditional would make it easier to merge all of this into master, if down the road we change our minds and decide that is better than keeping the branches separate.